listenbrainz/tests/integration/test_websockets.py

```diff
@@ -0,0 +1,40 @@
+import socketio
+import json
+import time
+
+from listenbrainz.tests.integration import ListenAPIIntegrationTestCase
+
+
+class WebSocketTests(ListenAPIIntegrationTestCase):
```

Comment:
I ran this locally and the test now works fine. I suggest you to update the asserts in the listen event to compare some fields in the listen sent and received.

Also, add another test for playing now.

Reply:
Relieved to know that the tests finally work now. Will work on the assert and playing test now